datadog_checks_dev/datadog_checks/dev/tooling/templates/check/{check_name}/tests/__init__.py

```diff
@@ -0,0 +1,3 @@
+# (C) Datadog, Inc. {year}
```

Comment:
Unless they're datadog employees we dont want to be forcing them to be signing over copyright to Datadog. In extras this could cause issues.

Reply:
@jeremy-lq what should we do here? while this is useful for dd employees, it's not necessarily something we want to do for other authors in Extras.